libretroshare/src/util/retrodb.cc

```diff
@@ -202,10 +202,8 @@ bool RetroDb::execSQL(const std::string &query){
             break;
         }
 
-        now = time(NULL);
-        delta = stamp - now;
-
-        if(delta > TIME_LIMIT){
+        if(time(NULL) > stamp + TIME_LIMIT)

```

Comment:
Shouldn't we have `rstime_t tTimeout = time(NULL) + TIME_LIMIT;` at line 188 and here compare with `if(time(NULL) > tTimeout)` so we avoid calculating the sum at iteration?

Reply:
You're right about the extra cost., but it's just an addition of integers, so it's totally negligible here.

BTW, before the change, stamp-now was always negative. Because delta is uint32_t, that meant delta>TIME_LIMIT was always true! Interestingly, after putting a printf there I realized that the condition was never ever checked.